charts/cluster-api-cluster-openstack/templates/workload.yaml

```diff
@@ -115,4 +123,6 @@ spec:
             node-labels: {{- include "openstack.nodelabels.workload" $context | nindent 14 }}
           taints:
             {{- include "openstack.taints.workload" $ | nindent 10 }}
+        patches:
```

Comment:
What happens when you provision something with no content?  I can see...
```
        // Content is the actual content of the file.
        // +optional
        Content string `json:"content,omitempty"`
```

So the content itself is optional, I just dunno if kubeadm is gonna crap out with an empty patch file.  Talk to me Goose!